src/sentry/integrations/pagerduty/client.py

```diff
@@ -3,6 +3,15 @@
 from sentry.utils import json
 from sentry.integrations.client import ApiClient
 from sentry.models import EventCommon
+from sentry.api.serializers import serialize, ExternalEventSerializer
+
+LEVEL_SEVERITY_MAP = {
```

Comment:
Is it worth adding a test that makes sure we're accounting for all levels we have for events?

I'm thinking of a case where we introduce a new level and then we just start sending `errors` to PD.

Reply:
I don't think there is any plan to add any new levels afaict

Reply:
Yea, fair. I guess if one does get added, we'll just default to "error", which doesn't seem too bad.